### ISSUE TITLE
Add ability to create attachments on issues

### ISSUE THREAD

**Issue by zhouxiaoli** [sumoheavy/jira-ruby#189]
how to create attachments, just like '.png' or 'file'
thanks!

**Comment by diwaniuk** [comment]
Hi,

try use JIRA::Resource::Attachment class

--- example code ---
```
attachment = JIRA::Resource::Attachment.new **@client**, issue: **issueobj**
attachment.save! 'file' => **file**
```

**Comment by Karamorf** [comment]
This would be nice to have ...

**Comment by VagishVela** [comment]
Can we not have a PR to implement this based off the code in the fork?

**Comment by SimonMiaou** [comment]
Was implemented in #244
